azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -663,3 +671,14 @@ def invoke_device_module_method(self, device_id, module_id, direct_method_reques
         return self.protocol.device_method.invoke_module_method(
             device_id, module_id, direct_method_request
         )
+
+    def send_c2d_message(self, device_id, message):
+        """Send a C2D mesage to a IoTHub Device.
+
+        :param str device_id: The name (Id) of the device.
+        :param str message: The message that is to be delievered to the device.
+
+        :raises: Exception if the Send command is not able to send the message
+        """
+
+        self.amqp_svc_client.send_message_to_device(device_id, message)
```

Comment:
I think an explicit return is missing here

Reply:
We are not returning anything here by design

---
In reply to: [380997235](https://github.com/Azure/azure-iot-sdk-python/pull/465#discussion_r380997235) [](ancestors = 380997235)